set_new_ip.yml

```diff
@@ -0,0 +1,58 @@
+---
+- hosts: nodes
+
+  vars_files:
+      - provision.yml
+
+  become: true
+
+  tasks:
+    - name: Find node's interface name                                                                                        
+      shell: | 
+        ip route | grep default | awk '{print $5;exit}'
+      register: node_interface_name
+
+    - name: "Configuring interface {{node_interface_name.stdout}}"                                                            
+      nmcli:
+        conn_name: "{{node_interface_name.stdout}}"                                                                             
+        ifname: "{{node_interface_name.stdout}}"
+        type: ethernet
+        state: present
+        autoconnect: yes
+        ip4: "{{ip}}"                                                                                                     
+        gw4: "{{gw}}" 
+      when: ansible_os_family == "RedHat"
+
+    - name: "Update Debian-based interfaces IP"
+      lineinfile:
+        dest: /etc/network/interfaces
+        regexp: "^(.*)169.254.250.250(.*)$"
+        line: "        address {{ip}}"
+      when: ansible_os_family == "Debian"
+
+    - name: "Update Debian-based interfaces gateway"
+      lineinfile:
+        dest: /etc/network/interfaces
+        regexp: '^(.*)169.254.251.251(.*)$'
+        line: '        gateway {{gw}}'
+      when: ansible_os_family == "Debian"
+
+    - name: "Remove auxilary Debian-based interfaces network attribute"
+      lineinfile:
+        dest: /etc/network/interfaces
+        regexp: '^(.*)network(.*)$'
+        state: absent
+      when: ansible_os_family == "Debian"
+
+    - name: "Remove auxilary Debian-based interfaces broadcast attribute"
+      lineinfile:
+        dest: /etc/network/interfaces
+        regexp: '^(.*)broadcast(.*)$'
+        state: absent
+      when: ansible_os_family == "Debian"
+
```

Comment:
We need to replace the install urls in `/etc/apt/sources.list`:

`- name: "Replace mirror URL in mirror list"
      lineinfile:
        dest: /etc/apt/sources.list
        regexp: '^(.*)network(.*)$'
        state: absent
      when: ansible_os_family == "Debian"`